### DISCUSSION TITLE
Zephyr-Hello World is not working

### DISCUSSION THREAD

**Discussion by tripathidivya** [zephyrproject-rtos/zephyr#52302] [Q&A]
Hello All, 

**Describe the bug**
I am trying to build a Zephyr Hello_world sample project as mentioned in this video: https://www.youtube.com/watch?v=vMDxmEwu51M

but I am facing issue there. So, I was looking for the solution and found one work around here: https://github.com/zephyrproject-rtos/zephyr/issues/40827

Describe the bug
Building any tensorflow lite example using VS Code would show an error in prj.conf :
CONFIG_TENSORFLOW_LITE_MICRO was assigned the value y, but got the value n. Missing dependencies: 0

**To Reproduce**

**Build any example under sample\modules\tensorflow**
Expected behavior
fatal error: tensorflow/lite/micro/all_ops_resolver.h: No such file or directory

**Impact**
Showstopper.

As mentioned in the link's comment section I have tried to follow the same steps but while updating west I have faced the issue like this for civetweb and tflite-micro
![image](https://user-images.githubusercontent.com/96341470/202237419-d4caba79-eafe-412c-9105-39be10e604ac.png)

and in the end it generated the empty folders for civetweb and for tflite-micro.

While updating it is asking for the github credentials:
![image](https://user-images.githubusercontent.com/96341470/202237465-77266ea8-b251-4d77-b074-b3fae84df1c8.png)

Also when I check on Zephyr-rtos repositry:
https://github.com/zephyrproject-rtos/zephyr/tree/main/modules

I have noticed there is no description in the zephyr-rtos repository for civetweb and for tflite-micro. it has only only these two files inside tflite-micro:
![image](https://user-images.githubusercontent.com/96341470/202237635-5d723f36-d610-47b4-bab7-de2b9827a1b6.png)

I have tried to copy this tflite folder into the nrf-sdk folder but the issue is still same.

**Environment (please complete the following information):**
OS: ubuntu(VM box 22.04)
Toolchain: nRF Connect SDK v2.1.2
Zephyr SDK v2.6.0 https://github.com/zephyrproject-rtos/zephyr/tree/zephyr-v2.6.0

Can you please help me to resolve this issue?

Regards,
Divya

**Comment by stephanosio** [comment]
Zephyr 2.6.0 is no longer supported. Please try with the LTS2 (2.7) or the latest version of Zephyr.

https://docs.zephyrproject.org/latest/develop/getting_started/index.html

**Reply by tripathidivya** [reply]
Hello @stephanosio, I have followed the link and luckily able to print hello world with zephyr project.
But now I want to test the sample code provided here: https://docs.zephyrproject.org/latest/samples/modules/tflite-micro/hello_world/README.html

and I have followed the steps mentioned and downloaded the latest Corstone-300 FVP from here:(https://developer.arm.com/tools-and-software/open-source-software/arm-platforms-software/arm-ecosystem-fvps).

After that when I tried to run this below command:
![image](https://user-images.githubusercontent.com/96341470/202416095-e502ee34-e267-48a2-a6db-0cd5272e5b5e.png)

and when I run this command then I have faced an error like this:
![image](https://user-images.githubusercontent.com/96341470/202418650-e541d881-51f5-4a7f-8c8b-20780774d0e3.png)
![image](https://user-images.githubusercontent.com/96341470/202418563-8d436b29-2af9-4e0c-b5d0-a93ad6dfd029.png)

Could you please guide what will be the solution for this issue?

Regards,
Divya

**Reply by yashi** [reply]
@tripathidivya, please press "Mark as answer" on @stephanosio 's, and create a new question.

**Comment by ivan2angelov** [comment]
I asked for help in Zephyr's tensorflow Discord https://discord.com/channels/720317445772017664/934458607859281930
I am getting exactly the error described by @tripathidivya
"CONFIG_TENSORFLOW_LITE_MICRO was assigned the value y, but got the value n. Missing dependencies: 0"
I have my own Zephyr project with my own board based on nRF5340. I am using Zephyr v3.2.99-ncs2, nrf-sdk is v2.3.0.
I modified NRF west.yml as suggested by @laurenmurphyx64 :
` git diff west.yml
diff --git a/west.yml b/west.yml
index a93676a6..5b16929e 100644
--- a/west.yml
+++ b/west.yml
@@ -66,6 +66,7 @@ manifest:
         #
         # Please keep this list sorted alphabetically.
         name-allowlist:
+          - tflite-micro
           - TraceRecorderSource
           - canopennode
           - chre`
I ran west update but I get no luck with adding CONFIG_TENSORFLOW_LITE_MICRO=y to my project.
When I ran west -v update I noticed following line:
west.manifest: project tflite-micro in file None ignored: an importing manifest blocked or did not allow it
Can someone give my tips what is going wrong?

**Comment by I77794** [comment]
Hi all,

I'm encountering an issue while trying to build a Zephyr project and would appreciate any help resolving it. The build process starts normally, but fails with the following error: 
![image](https://github.com/user-attachments/assets/226719e2-b024-440f-a5b5-d604a4e32017)
but I'm not sure how to resolve this.

Has anyone encountered this issue before or know what might be causing it?

Thanks in advance for your support!

Best regards,
Pavan
